whylog/config/__init__.py

```diff
@@ -48,8 +50,31 @@ def _save_rule_definition(self, rule_definition):
     def _save_parsers_definition(self, parser_definitions):
         pass
 
+    # mocked investigation plan for 003_match_time_range test
+    def mocked_investigation_plan(self):
+        matcher = RegexFilenameMatcher('localhost', 'node_1.log', 'default')
+        default_log_type = LogType('default', [matcher])
+        cause = RegexParser('cause', '^root cause$', [1], 'default', {1: 'date'})
```

Comment:
Please upgrade this regex (here and line below) according to changes in file 003_match_time_range/node_1.log